db/migrations/20220923124812_add_package_count.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+alter table repositories drop column package_count;
+
+COMMIT;
```

Comment:
The new column was added / removed with the migrations; tried by:

```sh
make db-clean clean build db-up
./release/dbmigrate down -step 1
make db-cli-connect
\d repositories
\q
./release/dbmigrate up -step 1
make db-cli-connect
\d repositories
\q
```
working as expected :+1: